simple-local-avatars.dev.js

```diff
@@ -96,4 +195,115 @@ function avatar_lock( lock_or_unlock ) {
 		avatar_form_button.attr('disabled','disabled');
 		avatar_spinner.show();
 	}
-}
\ No newline at end of file
+}
+
+function myTheme_calculateImageSelectOptions(attachment, controller) {
```

Comment:
Aside from the namespacing issue commented above, I think it makes sense to use `snake_case` for consistency with the rest of the functions defined in this file.